src/gleam/list.gleam

```diff
@@ -1013,11 +1013,11 @@ pub fn zip(xs: List(a), ys: List(b)) -> List(#(a, b)) {
 /// ```
 ///
 pub fn strict_zip(
-  l1: List(a),
-  l2: List(b),
+  list: List(a),
+  with other: List(b),
```

Comment:
This is an API change so could we have an entry in the changelog please 🙏

Reply:
Sure!